javascript/index.js

```diff
@@ -14,52 +15,122 @@ let milUni = document.getElementById('milUni');
 let splits = document.getElementById('splits');
 
 function printTime() {
-  // ... your code goes here
+  getMinutes(); //return minutes
```

Comment:
getMinutes and getSeconds are methods of the chronometer, so in order to call them you need to point to the chronometer. So: chronometer.getSeconds and chronometer.getMinutes